app/src/main/java/com/hedvig/app/feature/marketing/ui/MarketingActivity.kt

```diff
@@ -290,18 +293,37 @@ class MarketingActivity : BaseActivity() {
         login.show()
         getHedvig.show()
 
+        val market = getMarket()
+        if (market == null) {
+            e { "Programmer error: No market set when viewing ${this.javaClass.name}" }
+            return
+        }
```

Comment:
Should we open MarketPickerAcitvity here?

Reply:
Good idea! I think yes

Reply:
Actually, we've actually validated earlier in onCreate that we have a Market. Given that, I think that this is fine.